tests/msc2946_test.go

```diff
@@ -134,6 +143,14 @@ func TestClientSpacesSummary(t *testing.T) {
 			"via": []string{"hs1"},
 		},
 	})
+	// Note that this link gets ignored since R2 is not a space.
```

Comment:
Where do we verify that assumption?

Reply:
Near lines 213 - 219 below we do a `match.JSONCheckOff` call against `events` and do not include this.